library/Vanilla/Web/LegacyTwigViewHandler.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace Vanilla\Web;
+
+use Vanilla\Contracts\Web\LegacyViewHandlerInterface;
+
+/**
+ * Gdn_Controller view handler for twig.
+ */
+class LegacyTwigViewHandler implements LegacyViewHandlerInterface {
+    use \Garden\TwigTrait;
+
+    /**
+     * @inheritdoc
```

Comment:
Coding standards check doesn't like the first line of a method's doc block being a tag.

```suggestion
     * Render the given view.
```